src/server/scripts/Spells/spell_druid.cpp

```diff
@@ -1413,6 +1416,54 @@ class spell_dru_omen_of_clarity_restoration : public AuraScript
     }
 };
 
+// 383410 - Celestial Alignment
+// 390414 - Incarnation: Chosen of Elune
+class spell_dru_orbital_strike : public SpellScript
+{
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_DRUID_ORBITAL_STRIKE_TALENT, SPELL_DRUID_ORBITAL_STRIKE_DAMAGE });
+    }
+
+    bool Load() override
+    {
+        return GetCaster()->HasAura(SPELL_DRUID_ORBITAL_STRIKE_TALENT);
```

Comment:
Load is unneccessary - 383410 and 390414 are cast only when caster has SPELL_DRUID_ORBITAL_STRIKE_TALENT (390378 uses SPELL_AURA_OVERRIDE_ACTIONBAR_SPELLS, spell ids are selected by the server, they don't come from client)